src/components/OptionListContextProvider.tsx

```diff
@@ -229,7 +229,7 @@ function OptionsListContextProvider({children}: OptionsListProviderProps) {
 
         // This effect is used to update the options list when personal details change so we ignore all dependencies except personalDetails
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
-    }, [personalDetails]);
+    }, [personalDetails, reportAttributes?.reports]);
```

Comment:
As the comment mentioned above, this useEffect should only handle personalDetails and all other dependencies should be ignored. Adding the reportAttributes dependency is causing a major performance regression https://github.com/Expensify/App/pull/69637 due to frequent recomputation